apps/project/tests/test_apis.py

```diff
@@ -1087,3 +1091,17 @@ def test_project_entries_stats(self):
         response = self.client.get(url)
         self.assert_200(response)
         self.assertEqual(response.json()['status'], 'success')
+
+    def test_project_lead_groups_api(self):
+        project = self.create_project(role=self.normal_role, use_api=False)
+        lead_group1 = self.create(LeadGroup, project=project)
+        lead_group2 = self.create(LeadGroup, project=project)
+
+        url = f'/api/v1/projects/{project.pk}/lead-groups/'
+
+        self.authenticate()
+        response = self.client.get(url)
+        self.assert_200(response)
+        # Only provide projects leads-group [Pagination is done for larger dataset]
+        assert set([lg['id'] for lg in response.json()['results']]),\
```

Comment:
Here also. I  think it should be `assert val1 == val2`, not `assert val1, val2`.

Reply:
Fixed